CHANGELOG.md

```diff
@@ -88,7 +88,17 @@
 
 * [CHANGE] Ruler: changed ruler deployment max surge from `0` to `50%`, and max unavailable from `1` to `0`. #4381
 * [CHANGE] Memcached connections parameters `-blocks-storage.bucket-store.index-cache.memcached.max-idle-connections`, `-blocks-storage.bucket-store.chunks-cache.memcached.max-idle-connections` and `-blocks-storage.bucket-store.metadata-cache.memcached.max-idle-connections` settings are now configured based on `max-get-multi-concurrency` and `max-async-concurrency`. #4591
-* [CHANGE] Add support to use external Redis as cache. #4386
+* [CHANGE] Add support to use external Redis as cache. Following are some changes in the jsonnet config: #4386 #4640
```

Comment:
The selection of settings mentioned here seems random (some enabled flags but not others, some item size flags but not others). Can we say that the various `enabled` flags have been renamed and the `backend` flags have been added since that a bit more accurately describes the change.

Reply:
I was trying to be more specific about the exact changes that we did. Can reordering the items in the bullet list help? If not I will apply your suggestion.

Reply:
I simplified the changelogs detail as suggested.

Reply:
Updated with latest @pracucci suggestion.